lib/excon/constants.rb

```diff
@@ -104,7 +110,8 @@ module Excon
     :mock => 'Mock',
     :retries_remaining => 'Idempotent', # referenced in Instrumentor, but only relevant with Idempotent
     :retry_limit => 'Idempotent', # referenced in Instrumentor, but only relevant with Idempotent
-    :retry_interval => 'Idempotent'
```

Comment:
It is very minor/pedantic, but I generally alphabetize keys for things like this. Would you mind moving the retry_errors to the appropriate spot for that? I just find it can make debugging/rereading things smoother in the future (and doesn't usually take much more effort in the present).

Reply:
I'd ask the same thing for the other instances of the key below, but will just mention it once. Does that make sense? Thanks!

Reply:
I have alphabetized retry params where listed. However, i did notice that some of other params are not alphabetized but i really didn't want to touch them in this PR.

Reply:
Oops, the others must have slipped through on accident at some point. Thanks for fixing the new one.